packages/flutter_test/lib/src/binding.dart

```diff
@@ -1218,7 +1218,8 @@ enum LiveTestWidgetsFlutterBindingFramePolicy {
   /// which can cause differences in behavior.
   fullyLive,
 
-  /// Ignore any request to schedule a frame.
+  /// Ignore any request to schedule a frame. This is not benchmark for general
+  /// performance test but intended for drawing frames beyond engine requests.
```

Comment:
Nit: "This is not for integration performance tests" may be better than "This is not benchmark for general performance test"?

BTW, from the usages of `pumpBenchmark`, I had an impression that this is for benchmarking the UI thread frame build time only. This does not seem to be suitable for benchmarking the raster thread performance? CC @dnfield @Hixie for confirmation.

Reply:
This I'm now not sure. We have two placing for documenting this: 
- https://api.flutter.dev/flutter/flutter_test/LiveTestWidgetsFlutterBindingFramePolicy-class.html
- https://api.flutter.dev/flutter/flutter_test/LiveTestWidgetsFlutterBinding/framePolicy.html
I'll need to discuss further.

Reply:
Flutter style says that the first line should be a single sentence.

We should add any further explanations below the first line.

This also is a bit confusing to me, especially given the subsequent documentation below this - maybe we could just amend those lines to explain what types of benchmarks a user is interested in and which types this is meant for?

Reply:
I'm thinking if we should combine the dartdocs I listed in the above link and change one of them to a link. And it doesn't explicitly say it but I found `benchmark` flag prevents any `ticker` triggered animation (which can be inferred from it ignores framework from triggering new frame), which I think should be added to the doc.